app/uk/gov/hmrc/customs/declaration/logging/LoggingHelper.scala

```diff
@@ -61,7 +61,7 @@ object LoggingHelper {
         a.authorisedAs match {
           case Csp(badgeIdentifier, _) => s"[authorisedAs=Csp($badgeIdentifier)]"
           case NonCsp(eori, _) => s"[authorisedAs=NonCsp($eori)]"
```

Comment:
`authorisedAs=FileUploadCsp` looks wrong - this class has been renamed to `CspWithEori` (more generic)